aclogview/CM_Movement.cs

```diff
@@ -489,8 +492,8 @@ public class MovementEvent : Message {
             rootNode.Nodes.Add("style = " + style);
             TreeNode motionStateNode = rootNode.Nodes.Add("motionState__guessedname = ");
             motionState__guessedname.contributeToTreeNode(motionStateNode);
-            rootNode.Nodes.Add("stickToObject = " + stickToObject);
-            rootNode.Nodes.Add("moveToObject = " + moveToObject);
+            rootNode.Nodes.Add("stickToObject = " + stickToObject.ToString("X"));
+            rootNode.Nodes.Add("moveToObject = " + moveToObject.ToString("X"));
```

Comment:
A bunch of these (not just here) are still not going through a call to FormatGuid.